test/serializer_test.rb

```diff
@@ -70,4 +70,37 @@
       json_data.must_equal json_expected
     end
   end
+
+  context 'serialize singular record' do
+    let(:relation)   { Note.where(name: 'Title').first }
+    let(:controller) { NotesController.new }
+    let(:options)    { }
+
+    before do
+      @note = Note.create content: 'Test', name: 'Title'
+      @tag = Tag.create name: 'My tag', note: @note, popular: true
+    end
+
+    it 'generates the proper json output' do
+      # XXX Not sure why the key order is inverted here...
```

Comment:
I think this is, because it's not actually using the database serialization. We should alter the test to ensure the original serializer is not called.